### ISSUE TITLE
Prefix declarations per notebook

### ISSUE THREAD

**Issue by kvistgaard** [zazuko/vscode-sparql-notebook#67]
It would be nice to allow declaring prefixes per notebook.

When there are many prefixes, it would be neat to declare them all in one cell and let all other cells reuse them.

(I am currently working with a notebook where most queries use the same 20+ prefixes...)

**Comment by BenjaminHofstetter** [comment]
I fully agree that Prefix handling is something to improve. I use "copy cell down" very often to copy the prefixes. 

<img width="467" alt="Screenshot 2023-12-05 at 19 51 02" src="https://github.com/zazuko/vscode-sparql-notebook/assets/8033981/1c845315-b034-45a6-a237-0da79726c077">

I was thinking a prefix set per notebook might be a solution or "global" for all notebooks. 
It's under investigation. We have internal needs for something like this not related to sparql notebook.
